docker-compose.dev.yml

```
@@ -185,8 +190,83 @@ services:
       graph-db:
         condition: service_healthy
 
+  bh-authentik:
+    profiles:
+      - sso
+      - sso-only
+    image: ${AUTHENTIK_IMAGE:-ghcr.io/goauthentik/server}:${AUTHENTIK_TAG:-2023.10.7}
+    restart: unless-stopped
+    command: server
+    environment:
+      AUTHENTIK_REDIS__HOST: bh-authentik-valkey
+      AUTHENTIK_POSTGRESQL__HOST: bh-authentik-db
+      AUTHENTIK_POSTGRESQL__USER: ${BH_PG_USER:-authentik-bloodhound}
+      AUTHENTIK_POSTGRESQL__NAME: ${BH_PG_DB:-authentik-bloodhound}
+      AUTHENTIK_POSTGRESQL__PASSWORD: ${BH_PG_PASS:-bloodhoundcommunityedition}
+      AUTHENTIK_SECRET_KEY: ${BH_ATK_SECRET:-bloodhoundcommunityedition}
+    labels:
+      - traefik.enable=true
+      - traefik.http.routers.authentik.rule=Host(`${BH_AUTHENTIK_HOSTNAME:-authentik.localhost}`)
+      - traefik.http.routers.authentik.service=authentik
+      - traefik.http.services.authentik.loadbalancer.server.port=9000
+    ports:
+      - "${COMPOSE_PORT_HTTP:-9000}:9000"
+      - "${COMPOSE_PORT_HTTPS:-9443}:9443"
+    depends_on:
+      - bh-authentik-db
+      - bh-authentik-valkey
+
+  bh-authentik-worker:
+    profiles:
+      - sso
+      - sso-only
+    image: ${AUTHENTIK_IMAGE:-ghcr.io/goauthentik/server}:${AUTHENTIK_TAG:-2023.10.7}
+    restart: unless-stopped
+    command: worker
+    environment:
+      AUTHENTIK_REDIS__HOST: bh-authentik-valkey
+      AUTHENTIK_POSTGRESQL__HOST: bh-authentik-db
+      AUTHENTIK_POSTGRESQL__USER: ${BH_PG_USER:-authentik-bloodhound}
+      AUTHENTIK_POSTGRESQL__NAME: ${BH_PG_DB:-authentik-bloodhound}
+      AUTHENTIK_POSTGRESQL__PASSWORD: ${BH_PG_PASS:-bloodhoundcommunityedition}
+      AUTHENTIK_SECRET_KEY: ${BH_ATK_SECRET:-bloodhoundcommunityedition}
+    depends_on:
+      - bh-authentik-db
+      - bh-authentik-valkey
+
+  bh-authentik-valkey:
```

A bit pedantic but the services can likely stay as `authentik-XYZ` as they aren't bh services 
```suggestion
  authentik-valkey:
```